### ISSUE TITLE
Satellite crashing(ish)

### ISSUE THREAD

**Issue by CoreyJ87** [rhasspy/wyoming-satellite#78]
For some reason every now and then the satellite disconnects from home assistant and the only way to get it working again is to goto the wyoming satellite integration and restart it. Then it will pick me up again for a few minutes or so. Then just randomly stop reacting to the wake word again. Theres no errors in any of the logs on the PI. It just looks like its sitting there waiting like normal. Although, the wake word just stops working. Not sure if I should post this here or on the wake word repo. Thanks. This is so awesome regardless.

**Comment by MichaelBuczek** [comment]
I have the same problem. Apart from that the solution is - just as you are saying - awesome.

**Comment by jacobswe** [comment]
Yeah, I've noticed similar instability when making configuration changes to the services I have running on the satellite. I have a habit of doing a hard `sudo reboot` and then opening up the Devices & Services and reloading the device through the Wyoming integration. I wish there was a way to stream logs back or push config changes without the hassle of SSHing into the machine, but I'm sure it'll only get easier over time.

**Comment by debdalerichard** [comment]
I've had the same issue...I found replacing the SD card with a "ScanDisk High Endurance" Sd card( in 1 unit to test) solves most issues I've had the satellite running for a month with no problems
